lib/semian/unprotected_resource.rb

```diff
@@ -0,0 +1,31 @@
+module Semian
+  class UnprotectedResource
+    attr_reader :name
+
+    def initialize(name)
+      @name = name
+    end
+
+    def tickets
+      0
```

Comment:
`-1` perhaps? Just to distinguish completely cleanly in monitoring. `0` is possible in the semaphore, `-1` is completely impossible.